lib/switchtec.c

```diff
@@ -1333,36 +1333,64 @@ static int log_c_to_file(struct switchtec_dev *dev, int sub_cmd_id, int fd)
 	return 0;
 }
 
+static int log_ram_flash_to_file(struct switchtec_dev *dev,
+				 int subcmd1, int subcmd2, int subcmd3,
```

Comment:
Some better names for subcmd1, subcmd2 and subcmd3 would be appropriate.... subcmd_gen5, subcmd_flags, subcmd_legacy... or some such.